mindjunkies/accounts/tests.py

```
@@ -0,0 +1,56 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from mindjunkies.home.views import home
+from mindjunkies.courses.models import Course, Enrollment, CourseTeacher
+
+User = get_user_model()  # Use custom User model
+
+class HomeViewTest(TestCase):
+    def setUp(self):
+        """Set up test environment"""
+        self.client = self.client  # Use Django’s test client
```

Ruff (0.8.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)